tests/unit/tasks/test_sites.py

```diff
@@ -0,0 +1,33 @@
+import luigi
+import luigi.configuration
+
+import htetl.tasks.sites as tasks_sites
+import htetl.extract.sites as extract_sites
+from ... import test_data, test_utils
+
+
+def setup_module():
+    test_config = luigi.configuration.LuigiConfigParser.instance()
+    test_config.set('QueryPostgres', 'host', 'fake_host')
+    test_config.set('QueryPostgres', 'database', 'fake_db')
+    test_config.set('QueryPostgres', 'user', 'fake_user')
+    test_config.set('QueryPostgres', 'password', 'fake_password')
+
```

Comment:
Is setup_module used anywhere or is this ran automatically by luigi?

Reply:
`setup_module` is used by nose. A little more details here if you are interested http://nose.readthedocs.io/en/latest/writing_tests.html.